docs/2023/QuML/Enhanced QuML Player/index.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 1
+title: Introduction
+slug: /2023/QuML/Enhanced Quml Player for Enhanced Question types
```

Comment:
keep the slug same as the path of directory in which it is kept. See the directory name